### DISCUSSION TITLE
Dexcom ONE support

### DISCUSSION THREAD

**Discussion by kaloyansarafov** [NightscoutFoundation/xDrip#2021] [Q&A] [answered]
Dexcom ONE recently released in Bulgaria, and it looks to be almost the same as the G6, are there plans to add support for it?

It's currently the only sensor system available here, and it would be nice to be able to use it with AndroidAPS.

**Comment by Navid200** [comment, marked as answer]
> Dexcom ONE recently released in Bulgaria, and it looks to be almost the same as the G6, are there plans to add support for it?
> It's currently the only sensor system available here, and it would be nice to be able to use it with AndroidAPS.

https://navid200.github.io/xDrip/docs/Dexcom-One.html
